src/services/logger/DatadogLogger.ts

```diff
@@ -1,28 +1,28 @@
 import { datadogLogs } from '@datadog/browser-logs'
 
-import type { EnvVars } from '@/services/env/Env'
+import type Env from '@/services/env/Env'
 import type { ClientConfigInterface } from '@/store/modules/config/config.types'
 
-type Env = (
-  key: keyof Pick<EnvVars,
-  'KUMA_PRODUCT_NAME'
-  >
-) => string
 export default class DatadogLogger {
-  env: Env
-  constructor(env: Env) {
+  env: Env['var']
+
+  constructor(env: Env['var']) {
     this.env = env
   }
 
   setup(config: ClientConfigInterface) {
     if (config.reports.enabled) {
+      const service = this.env('KUMA_PRODUCT_NAME').toLowerCase().replaceAll(' ', '-') + '-ui'
+
       datadogLogs.init({
-        clientToken: 'pub94a0029259f79f29a5d881a06d1e9653',
+        // The current client token is called “kuma-ui” in the Datadog UI.
+        // Previous versions of the application used a different client token attached to a different Datadog instance. This client token was called “kuma-gui” in the Datadog UI.
+        clientToken: 'pub1aadd2cab84c05bf9959e00a35b213a1',
         site: 'datadoghq.com',
         forwardErrorsToLogs: true,
-        service: this.env('KUMA_PRODUCT_NAME'),
         sampleRate: 100,
-        env: 'production', // logging is currently disabled in anything other than production
+        service,
+        env: import.meta.env.MODE,
```

Comment:
**Comment**: We’re not actually setting `NODE_ENV` anymore so I’m using `MODE` from Vite here.

Reply:
Yeah, I think it’s more safe to use a variable here:

- When you test out such an integration locally, it’s nicer to label events sent to Datadog with a dev environment
- If we were to change our approach to also log events from dev environments, we wouldn’t need to remember changing this